components/clp-package-utils/clp_package_utils/scripts/decompress.py

```
@@ -5,7 +5,12 @@
 import sys
 from typing import Optional
 
-from clp_py_utils.clp_config import CLPConfig, StorageEngine, StorageType
+from clp_py_utils.clp_config import (
+    CLP_DEFAULT_DATASET_NAME,
+    CLPConfig,
+    StorageEngine,
+    StorageType,
+)
```

Pylint (3.3.7)

[error] Unable to import 'clp_py_utils.clp_config'

(E0401)